src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/AcknowledgeClaimCallbackHandlerTest.java

```diff
@@ -240,68 +221,6 @@ void shouldReturnNoError_whenSoleTraderDateOfBirthIsInThePast() {
         private LocalDateTime newDeadline;
         private LocalDateTime acknowledgementDate;
 
-        @BeforeEach
-        void setup() {
-            newDeadline = LocalDateTime.now().plusDays(14);
-            when(deadlinesCalculator.plus14DaysAt4pmDeadline(any())).thenReturn(newDeadline);
-            acknowledgementDate = LocalDateTime.now();
-            when(time.now()).thenReturn(acknowledgementDate);
-        }
-
-        @Test
-        void shouldSetNewResponseDeadlineAndUpdateBusinessProcess_whenInvoked() {
-            CaseData caseData = CaseDataBuilder.builder().atStateNotificationAcknowledged().build().toBuilder()
-                .respondent1Copy(PartyBuilder.builder().individual().build())
-                .build();
-            CallbackParams params = callbackParamsOf(caseData, ABOUT_TO_SUBMIT);
-
-            var response = (AboutToStartOrSubmitCallbackResponse) handler.handle(params);
-
-            assertThat(response.getData())
-                .extracting("businessProcess")
-                .extracting("camundaEvent")
-                .isEqualTo(ACKNOWLEDGE_CLAIM.name());
-
-            assertThat(response.getData())
-                .extracting("businessProcess")
-                .extracting("status")
-                .isEqualTo("READY");
-
-            assertThat(response.getData())
-                .containsEntry("respondent1ResponseDeadline", newDeadline.format(ISO_DATE_TIME))
-                .containsEntry("respondent1AcknowledgeNotificationDate", acknowledgementDate.format(ISO_DATE_TIME));
-        }
-
-        @Test
-        void shouldCopyRespondentPrimaryAddresses_whenInvoked() {
-            CaseData caseData = CaseDataBuilder.builder()
-                .atStateRespondentFullDefenceAfterNotificationAcknowledgement()
-                .respondent2(PartyBuilder.builder().individual().build())
-                .addRespondent2(YES)
-                .build();
-            String address = "test address";
-            var expectedAddress = AddressBuilder.defaults().addressLine1(address).build();
-            caseData = caseData.toBuilder()
-                .respondent1Copy(caseData.getRespondent1().toBuilder().primaryAddress(expectedAddress).build())
-                .respondent2Copy(caseData.getRespondent2().toBuilder().primaryAddress(expectedAddress).build())
-                .build();
-            CallbackParams params = callbackParamsOf(V_1, caseData, ABOUT_TO_SUBMIT);
```

Comment:
This seems to be invoking V1 version for test. Should we not retain it under the AboutToSubmitCallbackV1 class below

Reply:
you're right, I deleted the entire nested class, I'll retain this one.